### ISSUE TITLE
Archive analytics archive to an archiving service (e.g. zenodo) periodically?

### ISSUE THREAD

**Issue by minrk** [jupyterhub/team-compass#504]
### Proposed change

Should we submit the mybinder.org analytics archive periodically (e.g. annually) to something like zenodo? It's inexpensive to serve it from the GCP project, but it does cost money and will go away eventually if the main GCP instance has to shut down. It might be nice to have annual archives somewhere more resilient.

It's pretty small (2.7GB for 2021), and can be an order of magnitude smaller if we reserialize 365 .jsonl files to e.g. feather or parquet.

I've been downloading it locally and [repacking to feather](https://gist.github.com/4d77b6129d3e24e75303040bf9c85b80) for easier analytics for a few things.

### Alternative options

Don't do this.

### Who would use this feature?

Folks interested in binder data on the long term. Could be citable, since it gets a DOI, etc.

### (Optional): Suggest a solution

1. at the end of each year, upload the year's data to zenodo or some other archive service
2. (optionally) reserialize .jsonl to feather or parquet for more efficient (we could also do this on the archive itself)
3. (optionally) include aggregations (also could be on the archive itself)

*(archive, archive, archive)*

**Comment by manics** [comment]
Definitely a good idea to archive it _somewhere_.

I haven't used Zenodo for storing/fetching data so can't say anything about it.... Does it support reading/streaming data directly from URLs?

**Comment by minrk** [comment]
> Does it support reading/streaming data directly from URLs?

I don't think so, it's usually a tarball. I'm not sure if there are other 'data archiving' services that are more suitable to particular data formats.

Since our data is small, a single download for a year (especially if reserialized) is not a lot to ask. Downloading and loading 3 years of historical data in feather format is usually going to be faster than just loading a month of .jsonl files already on disk.
